core/drivers/crypto/stm32/common.h

```diff
@@ -8,7 +8,42 @@
 
 #include <tee_api_types.h>
 
+enum stm32_cipher_ip_id {
+	CRYP_IP,
+	SAES_IP,
+};
+
+/*
+ * Macro to manage bit manipulation when we work on a local variable
+ * before writing only once to the hardware register.
+ */
+#define CLRBITS(v, bits)		((v) &= ~(bits))
+#define SETBITS(v, bits)		((v) |= (bits))
+#define CLRSETBITS(v, mask, bits)	(*(v) = (*(v) & ~(mask)) | (bits))
+
+#define TOBE32(x)			TEE_U32_BSWAP(x)
```

Comment:
We already have `TEE_U32_TO_BIG_ENDIAN()` and `TEE_U32_FROM_BIG_ENDIAN()`. They are perhaps a bit long, but until we add new more convenient generic macros that's what we use.

Reply:
Ok update in SAES and CRYP driver. I also added a commit to update authenc.c : https://github.com/OP-TEE/optee_os/pull/6168/commits/4c74bce77e568739678b6f64d0773c0aff400213